### PR TITLE
add-wheel-guide

### PR DESCRIPTION
**Pull request by JustinaPetr**
# Wheel docs guide - #ADOPT-1200

## Description

Add wheel docs guide

### CODE REVIEW

**Comment by JustinaPetr** [comment]
@3avi Before I spend more time polishing, wanted to get your thoughts on this structure.

One thought I have - would it make more sense to move `Wheel reference` as a subpage to Guides->Running Locally in favour of keeping Getting Started docs lean and stress-free?

File: website/versioned_docs/version-0.4.x/wheel-reference.mdx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+specify a custom one or skip this step if you don’t want to store a DID private key on a local file by pressing esc on your keyboard.
+
+### CAS URL
+`Dev`, `Clay` and `Mainnet` projects run a node that connects to CAS (Ceramic Anchor Service) to create anchor
+commits on the blockchain for the data streams generated for your project. You will be given an option  to specify CAS URL - you can 
+use the default suggestion (recommended in most of the cases) or specify a custom url if you run your own anchor service. 
+
+
+### CAS Authentication
+In order to control the nodes connected to CAS (Ceramic Anchor Service), you will have to configure the authentication. This will allow
```
Comment (dbcfd) [inline website/versioned_docs/version-0.4.x/wheel-reference.mdx:69]: Link to page on what CAS auth means.

File: website/versioned_docs/version-0.4.x/set-up-your-environment.mdx [hunk 2]
```diff
@@ -5,9 +5,114 @@ import TabItem from '@theme/TabItem'
...
+```
+
+Once Wheel is downloaded you are good to start configuring your project working directory. To kick it off, run the command below:
+
+```bash
+./wheel
+```
+
+This command will start a prompt in your command line allowing you to configure your entire working environment - from what Ceramic dependencies
+ you’d like to install to how your Ceramic node should be configured.
```
Comment (dbcfd) [inline website/versioned_docs/version-0.4.x/set-up-your-environment.mdx:90]: Should we mention something about help messages for prompts?

**Review by dbcfd — CHANGES_REQUESTED** [review]

**Review by 3avi — APPROVED** [review]
besides danny's comments to include references to other docs pages like CAS, content looks good to me Justina, thanks for updating the sidebar + adding the wheel reference subpage.

**Review by dbcfd — DISMISSED** [review]

**Review by dbcfd — APPROVED** [review]

**Pull request merged**